package exceptional_handling;

public class Throw_Exception {

	public static void main(String[] args) throws ArrayIndexOutOfBoundsException {
		//throw new NullPointerException();
		//throw new NullPointerException("Hey element is not holding any value");
        
		throw new ArrayIndexOutOfBoundsException();
		
	}

}
